src/Collections/ArrayGeneric.php

```diff
@@ -107,7 +107,19 @@ public function valid(): bool
     public function __debugInfo(): array
     {
         return [
-            'data' => $this->data
+            'data' => $this->data,
```

Comment:
You added comma after last element in array:
```php
return [
    'data' => $this->data,
];
```
for sure to omit git changes when you add next element (?) but in other place you align variables and values:
```php
$type              = ltrim($input->getArgument('type'),'\\');
$namespace         = $input->getArgument('namespace');
$genericCollection = new GenericCollection($type, $namespace);
```
so when you add new variable with longer name then last one you will get more git changes.

Reply:
IDE is just configured to do this